frontend/src/Pages/RecruitmentApplicationsOverviewPage/components/WithdrawnApplications/WithdrawnApplications.tsx

```diff
@@ -0,0 +1,67 @@
+import { useQuery } from '@tanstack/react-query';
+import { useTranslation } from 'react-i18next';
+import { Link, Table } from '~/Components';
+import { getWithdrawnRecruitmentApplicationsApplicant } from '~/api';
+import type { RecruitmentApplicationDto } from '~/dto';
+import { KEY } from '~/i18n/constants';
+import { reverse } from '~/named-urls';
+import { ROUTES } from '~/routes';
+import { dbT } from '~/utils';
+import type { ApplicantApplicationManagementQK } from '../../RecruitmentApplicationsOverviewPage';
+import styles from './WithdrawnApplications.module.scss';
+
+type WithdrawnApplicationsProps = {
+  recruitmentId: string;
+  queryKey: ApplicantApplicationManagementQK;
+};
+export function WithdrawnApplications({ recruitmentId, queryKey }: WithdrawnApplicationsProps) {
+  const { t } = useTranslation();
+
+  const { data: withdrawnApplications = [] } = useQuery({
+    queryKey: queryKey.withdrawnApplications(recruitmentId),
+    queryFn: () => getWithdrawnRecruitmentApplicationsApplicant(recruitmentId).then((response) => response.data),
```

Comment:
Endre på getWithdrawnRecruitmentApplicationsApplicant til å returnere data, og så:

```suggestion
    queryFn: () => getWithdrawnRecruitmentApplicationsApplicant(recruitmentId),
```

Reply:
https://github.com/Samfundet/Samfundet4/pull/1595/commits/b92b70c96f8a4d07e50383f5fce1686d8b4d201d